eng/pipelines/templates/scripts/replace-relative-links.yml

```diff
@@ -0,0 +1,149 @@
+parameters: 
+  TargetFolder: ''
+  RootFolder: ''
+  BuildSHA: ''
+  RepoId: ''
+
+steps:
+  - task: PythonScript@0
+    displayName: Replace Relative Readme Links with Absolute References
+    inputs:
+      scriptSource: inline
+      script: |
+        import argparse
```

Comment:
While I'm good with this for now I do worry about the size of this inline script and how we plan to maintain this vs the py script itself.

Reply:
You're right. I don't really have a good response right now :(

Reply:
Perhaps we put this out as a release, make the yaml _download_ that release, unzip, and call it?

Reply:
Nothing to worry about right now but something to keep in the back of our mind to consider for clean-up especially if we end needing to make various changes to these scripts.